be/src/vec/sink/vdata_stream_sender.h

```diff
@@ -261,5 +274,24 @@ class VDataStreamSender::Channel {
     size_t _capacity;
     bool _is_local;
 };
+
+template <typename Channels, typename HashVals>
+Status VDataStreamSender::channel_add_rows(Channels& channels, int num_channels, const HashVals& hash_vals, int rows, Block* block) {
+    std::vector<int> channel2rows[num_channels];
```

Comment:
same to the up,  we should reuse the mem

Reply:
channel2rows is used for aggregate rows for channels, it's a temporary data served for the function of channel_add_rows, it's not used in other member functions, it's not has getter/setter, it's not visiable to outer, so i don't think make it as member data is a good  idea.

for performance, if you can prove it with rewriting code, please tell me